website/blog/2020-02-16-async.md

```diff
@@ -23,6 +23,8 @@ The latest MUnit v0.5.0 release fixes this issue by adding support for async
 tests on all platforms meaning that the example test case above fails as
 expected after 100 milliseconds.
 
+<!-- truncate -->
```

Comment:
so that this blog post doesn't take a huge amount of vertical space in https://scalameta.org/munit/blog/